src/libs/actions/User.js

```diff
@@ -53,7 +53,13 @@ function changePasswordAndNavigate(oldPassword, password) {
         .then((response) => {
             if (response.jsonCode !== 200) {
                 const error = lodashGet(response, 'message', 'Unable to change password. Please try again.');
-                Onyx.merge(ONYXKEYS.ACCOUNT, {error});
+                const errorCode = lodashGet(response, 'jsonCode', 0);
+                if (errorCode === 401) {
+                    Onyx.merge(ONYXKEYS.ACCOUNT, {error, errorType: 'current'});
+                }
+                if (errorCode === 666) {
+                    Onyx.merge(ONYXKEYS.ACCOUNT, {error, errorType: 'generic'});
```

Comment:
Move the current and generic to CONST.js

Reply:
Applied changes.